sdks/python/apache_beam/io/fileio.py

```diff
@@ -129,9 +129,10 @@ def process(self, file_metadata):
                 if isinstance(file_metadata, (str, unicode))
                 else file_metadata)
 
-    if metadata.path.endswith('/') and self._skip_directories:
+    if ((metadata.path.endswith('/') or metadata.path.endswith('\\'))
```

Comment:
Can you use `os.sep` (https://docs.python.org/3.3/library/os.html?highlight=os.sep#os.sep) ?

Reply:
I'd like to check for both, as we may want to access GCS files in windowss, which use `/`.